src/main/java/com/exactpro/epfast/template/DecimalField.java

```diff
@@ -0,0 +1,13 @@
+package com.exactpro.epfast.template;
+
+public interface DecimalField extends Instruction {
```

Comment:
We need two type of decimal fields: one which has only field operator and another which has separate definitions for mantissa and exponent